src/vivarium_census_prl_synth_pop/components/household.py

```diff
@@ -135,13 +148,41 @@ def on_time_step(self, event: Event):
 
         max_household_address_id = households["address_id"].max()
 
-        (pop, _, _,) = update_address_id_for_unit_and_sims(
+        (pop, self.households, _,) = update_address_id_for_unit_and_sims(
             pop,
-            moving_units=households,
+            moving_units=self.households,
             units_that_move_ids=movers["household_id"],
             starting_address_id=max_household_address_id + 1,
             unit_id_col_name="household_id",
             address_id_col_name="address_id",
         )
 
         self.population_view.update(pop)
+
+    ##################
+    # Helper methods #
+    ##################
+
+    def generate_initial_households(self, pop_data: SimulantData) -> pd.DataFrame():
+        households = self.population_view.subview(["household_id", "housing_type"]).get(
```

Comment:
Is there not a way to get housing type without looking at the housing_type column? If not, you're going to have trouble when you try to delete it. Can't we just define the GQ households explicitly and then set the rest of them as "Standard"?

Reply:
Oh I was just generating a new `initialized_housing_type` column that was used here instead as well as in fertility for newborns. I like your idea more if it works in fertility.